Testing/functional/tests/cypress/integration/ui/services/menu.js

```diff
@@ -0,0 +1,68 @@
+const { AuthMethod } = require("../../../support/constants");
+
+function toggleServices(enabled = true) {
+    cy.configureSettings({
+        services: {
+            enabled: enabled,
+        },
+    });
+}
+
+function login(isMobile) {
+    if (isMobile) {
+        cy.viewport("iphone-6"); // Set viewport to 375px x 667px
+    }
+    cy.login(
+        Cypress.env("keycloak.username"),
+        Cypress.env("keycloak.password"),
+        AuthMethod.KeyCloak
+    );
+    cy.checkTimelineHasLoaded();
```

Comment:
You can simplify this by removing the call to `checkTimelineHasLoaded` and specifying the 4th parameter of .login() to load a different page instead of the timeline, e.g. "/home".